### PR TITLE
Release frost-text into the wild

### PR DESCRIPTION
**Pull request by sandersky**
#MINOR#

### CODE REVIEW

File: addon/styles/addon.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import 'node_modules/ember-frost-css-core/scss/frost-core';
+@import 'node_modules/ember-frost-theme/scss/frost-theme';
+
+$border: 1px solid $frost-color-lgrey-5;
```
Comment (sglanzer) [inline addon/styles/addon.scss]: Good stuff

File: ember-cli-build.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+var EmberApp = require('ember-cli/lib/broccoli/ember-addon')
+
+module.exports = function (defaults) {
+  var app = new EmberApp(defaults, {
+    'ember-cli-mocha': {
+      useLintTree: false
+    },
+    sassOptions: {
+      includePaths: [
+        'bower_components/bourbon/app/assets/stylesheets',
```
Comment (sglanzer) [inline ember-cli-build.js]: Drop bourbon

File: bower.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "ember-frost-text",
+  "dependencies": {
+    "bourbon": "^4.2.6",
```
Comment (sglanzer) [inline bower.json]: Drop bourbon

File: tests/dummy/app/styles/app.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+@import 'bourbon';
```
Comment (sglanzer) [inline tests/dummy/app/styles/app.scss]: Bourbon

File: tests/dummy/config/environment.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    environment: environment,
+    baseURL: '/',
+    locationType: 'hash',
+    EmberENV: {
+      FEATURES: {}
+    },
+    APP: {}
+  }
+
+  switch (environment) {
```
Comment (sglanzer) [inline tests/dummy/config/environment.js]: Not sure why they didn't use `switch` before, this is an improvement

**Comment by sglanzer** [comment]
Approved LGTM

File: tests/dummy/config/environment.js (hunk 5 above)
Comment (sandersky) [inline tests/dummy/config/environment.js]: Or even `else if` for that matter.

**Pull request merged**